### PR TITLE
Docs

### PR DESCRIPTION
**Pull request by acmenezes**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,42 @@
...
+# podtracer
+
+podtracer is a cli tool inspired in the Linux command line utility called nsenter. nsenter is capable of running programs in selected Linux namespaces taking as input the file paths for the namespaces or process ids from where it derives the namespaces.
+
+podtracer does the same but taking as input pod names and kubernetes namespaces in order to run Linux tools against the pods as targets. That enables tools such as tcpdump, iperf, tc, ip and others to be used against pods and containers directly without the itermediary process of finding their respective process IDs and subsequent namespace file paths.
+
+It's designed to be run by an automated processes such as [snoopy-operator](https://github.com/fennec-project/snoopy-operator) that brings scalability to the next level. It's done by running podtracer as a container entry point for kubernetes jobs. Multiple jobs running tcpdump, for example, can capture packets on multiple pods at the same time and send the extracted data to a central data processing server.
+
+The tool is also intended to be used as troubleshooting tool in a stand alone mode to inspect speficic containers inside a Kubernetes platform.
+
```
Comment (skattoju4) [inline README.md:10]: should mention something about privileges and openshift vs vanilla kubernetes

File: README.md [hunk 2]
```diff
@@ -1,2 +1,42 @@
...
+# Build and Install
+
+There is no tagged release for podtracer at this point in the development but it can be built with go 1.15 or later like below. It's important to notice that at this point it doesn't support other operating systems other than Linux. It's under our plan to have a stand alone release for MacOS users reaching out to remote k8s clusters.
+
```
Comment (skattoju4) [inline README.md:14]: should have a requirements or dependencies section ex: dependency on cri-o

File: README.md (hunk 1 above)
Comment (acmenezes) [reply README.md:20]: Just pushed the requirements section. Thanks @skattoju .

File: README.md [hunk 3]
```diff
@@ -1,2 +1,55 @@
-# snoop
-Command line tool interfacing network packet captures and Linux namespaces for containers
+# podtracer
+
+podtracer is a cli tool inspired in the Linux command line utility called nsenter. nsenter is capable of running programs in selected Linux namespaces taking as input the file paths for the namespaces or process ids from where it derives the namespaces.
+
+podtracer does the same but taking as input pod names and kubernetes namespaces in order to run Linux tools against the pods as targets. That enables tools such as tcpdump, iperf, tc, ip and others to be used against pods and containers directly without the itermediary process of finding their respective process IDs and subsequent namespace file paths.
+
+It's designed to be run by automated processes such as [snoopy-operator](https://github.com/fennec-project/snoopy-operator) that brings scalability to the next level. It's done by running podtracer as a container entry point for kubernetes jobs. Multiple jobs running tcpdump, for example, can capture packets on multiple pods at the same time and send the extracted data to a central data processing server.
```
Comment (skattoju) [inline README.md:7]: think you should explicitly say that you can run commands across multiple pods targeted by a label

File: README.md (hunk 1 above)
Comment (skattoju) [reply README.md:20]: lgtm +1

File: README.md (hunk 2 above)
Comment (skattoju) [reply README.md:24]: fixed above

**Pull request merged**